scenarios/kubernetes_build.py

```diff
@@ -155,5 +155,7 @@ def main(args):
         '--extra-publish-file', help='Additional version file uploads to')
     PARSER.add_argument(
         '--allow-dup', action='store_true', help='Allow overwriting if the build exists on gcs')
+    PARSER.add_argument(
+        '--push-build', default='../release/push-build.sh', help='location of push-build.sh')
```

Comment:
nit: make this `--push-build-script` or similar? `--push-build` implies enabling pushing to me :man_shrugging:

Reply:
alternate bikeshed: make this `--release-repo` or something and then prepend that to `push-build.sh`.

Reply:
@ixdy but then your repo can't bikeshed the script name? :rofl:

Reply:
@ixdy well... the issue is we use a relative path here... I don't want to use absolute path and break everything?

Reply:
or... `--release-repo=../release`? :man_facepalming:

Reply:
shedded to `--push-build-script` :man_shrugging: